packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -4228,6 +4276,24 @@ void main() {
   });
 }
 
+class ColumnSelectionContainerDelegate extends MultiStaticSelectableSelectionContainerDelegate {
+  /// Copies the selected contents of all [Selectable]s, separating their
+  /// contents with a new line.
```

Comment:
Oh was this PR prompted by the issue about putting a newline between copied widgets?

Reply:
Yes, and anyone trying to use `SelectionContainer` has to re-implement or copy and paste a lot of this logic since it requires a `SelectionContainerDelegate`.